.github/workflows/enforce-license-compliance.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
This one is not needed

Reply:
```suggestion
```

Reply:
I was trying to see how much yamllint errors bother you :smile: 

Looks like it does not!